src/en/wizden-staff/maintainer/review-procedure.md

```diff
@@ -1,232 +1,176 @@
 
-For design documents proposing new features to be added to the game, the normal 2-approval system is used. 
-Once it is approved, however, it should not be merged and instead marked with the `S: Doc Approved` label. 
-This lets the author know that the design has initial approval, and they are good to start on the content PR. 
+While all PRs benefit from feedback and may be changed at a later time, Experimental PRs are intended to indicate to the community that feedback is especially appreciated.
 
-Once the content-side implementation is ready to be merged, ensure that the doc PR is updated to reflect relevant information and changes to the content PR.
-Afterward, both PRs can be merged at once.
+Adding the label to a PR will create a forum thread with the PR's name and number in the Feedback category of the forum. Once merged, any changelog entry for the PR will have an additional :test_tube: symbol to indicate its Experimental status.
 
-### Other
-
-For PRs which update instructional or reference documentation (_including but not limited to setup guides, style guides, system documentation_), only a single approval is needed before it can be merged.
-Likewise, only a single maintainer needs to express concern to close it.
-
-Note that this does not apply to changes to internal procedure or other modifications which require voting or group deliberation.
-
-
-## Review Guidelines
-The following are general guidelines to follow when reviewing PRs. 
-These are especially important to follow for first-time contributors.
-
-- Use **constructive** language and avoid being overly negative.
-
-
-- Try not to be _overly_ vulgar or swear in a derogatory way.
-
-
-- Avoid comments that personally criticize the author.
-It's okay to critique code and decisions, but you shouldn't insult their character.
-
-
-- For people who may not be familiar with a system, try to be more thorough in your explanations.
+It is strongly recommended that a `FeedbackPopup` prototype for the thread is added to `Resources/Prototypes/FeedbackPopup/feedbackpopups.yml`.
 
+### Merging PRs
+A pull request is ready to be merged if it fulfills the approval requirements associated with the PR, and all requested changes are resolved.
 
-- When possible, direct people to either the [developer wiki](https://docs.spacestation14.com/index.html), `#howdoicode` discord channel, or alternate source for information.
+Next:
+1. Check one final time to ensure that the changelog is correct and the breaking changes section is accurate.
+2. Add the pull request to the merge queue.
+3. Post the breaking changes announcement in the [Breaking Changes](https://forum.spacestation14.com/c/development/breaking-changes/70) category in `Development` on Discourse.
 
+If the pull request gets kicked out of the merge queue due to failing tests, check if the test fail is related to the PR. If not, re-add the pull request to the merge queue.
```

Comment:
This would be a good place to mention that we have a megaissue for tracking known Heisentests, which hopefully makes each one get fixed faster
https://github.com/space-wizards/space-station-14/issues/41081